apps/prairielearn/src/pages/instructorAssessmentAccessOverrides/instructorAssessmentAccessOverrides.sql

```diff
@@ -0,0 +1,115 @@
+-- BLOCK select_assessment_access_policies
+SELECT
+  format_date_full_compact (aap.created_at, coalesce(ci.display_timezone, c.display_timezone)) AS created_at,
+  aap.created_by::text AS created_by,
+  aap.credit::text AS credit,
+  format_date_full_compact (aap.end_date, coalesce(ci.display_timezone, c.display_timezone)) AS end_date,
+  aap.note AS note,
+  format_date_full_compact (aap.start_date, coalesce(ci.display_timezone, c.display_timezone)) AS start_date,
+  (
+    SELECT
+      name
+    FROM
+      groups
+    WHERE
+      id = aap.group_id
+  ) AS group_name,
+  (
+    SELECT
+      uid
+    FROM
+      users
+    WHERE
+      user_id = aap.user_id
+  ) AS student_uid,
+  aap.id AS id
+FROM
+  assessment_access_policies AS aap
+  LEFT JOIN course_instances AS ci ON (ci.id = $course_instance_id)
+  LEFT JOIN pl_courses AS c ON (c.id = $course_id)
```

Comment:
Rather than passing these in explicitly, we typically join through other tables to get to these. That is:

```suggestion
  JOIN assessments AS a ON (a.id = aap.assessment_id)
  JOIN course_instances AS ci ON (ci.id = a.course_instance_id)
  JOIN pl_courses AS c ON (c.id = ci.course_id)
```

This is untested, but it should get across the gist of what we're looking for.

Reply:
Fixed in d62fa40c8